### PR TITLE
 Feature/feature readme

### PR DESCRIPTION
**Pull request by nicoxx**
-motivation
-install
-contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,84 @@
...
 
-# gator
\ No newline at end of file
+# Gator Life!
+
+This is the codebase of [gator.life](http://www.gator.life)
+
+*WORK IN PROGRESS*
+
+Crawl the web and use machine-learning algorithms to provide a personalized list of links.
```
Comment (m-mokhtari) [inline README.md]: I think "machine-learning" should be writter without the "-".

File: README.md [hunk 2]
```diff
@@ -1,4 +1,84 @@
...
+
+Crawl the web and use machine-learning algorithms to provide a personalized list of links. 
+
+## Mission
+
+Whatever your passions are, today, a few gems will appear on the web. How to find them among billions of blog posts, videos and news articles?
+
+Gator.life aims to exlpore the web and provide the best content to each person.
+
+### Doesn't it exist already?
```
Comment (m-mokhtari) [inline README.md]: I would have wrote "Doesn't it already exist ?".

File: README.md [hunk 3]
```diff
@@ -1,4 +1,84 @@
...
+
+*WORK IN PROGRESS*
+
+Crawl the web and use machine-learning algorithms to provide a personalized list of links. 
+
+## Mission
+
+Whatever your passions are, today, a few gems will appear on the web. How to find them among billions of blog posts, videos and news articles?
+
+Gator.life aims to exlpore the web and provide the best content to each person.
```
Comment (m-mokhtari) [inline README.md]: Typo: explore

File: README.md [hunk 4]
```diff
@@ -1,4 +1,84 @@
...
+
+### Doesn't it exist already?
+
+Sort of, you have to trust closed platforms and their algorithms to do the job for you. This raises fundamental issues:
+
+Hidding behind to need to learn about you for their algorithms to work, Those platforms track always more of what you do. What do they know about you? What do they do with it?
```
Comment (m-mokhtari) [inline README.md]: Typo: Hiding.
I would have written: "behind THE need to ..."

File: README.md [hunk 5]
```diff
@@ -1,4 +1,84 @@
...
+
+When they choose something for you, is it what YOU want to see, or what THEY want you to see? Is it the most interesting content or the most monetizable? Who decides to promote or censor?
+
+### How is *Gator life* different?
+
+One word: **OPEN**. We believe it changes everything. Because it leads to another one: **ACCOUTABLE**.
```
Comment (m-mokhtari) [inline README.md]: Do you mean ACCOUNTABLE ?

File: README.md [hunk 6]
```diff
@@ -1,4 +1,84 @@
...
+
+You can judge if we are true to our mission. You can challenge every change and its rational. You can fill a issue or make a pull request each time you think we can do better.
```
Comment (m-mokhtari) [inline README.md]: Typo: rationale

**Comment by nicoxx** [comment]
ping: @m-mokhtari . I took you remarks into account and added a architecture section.

**Pull request merged**